code/game/machinery/exp_cloner.dm

```diff
@@ -7,6 +7,8 @@
 	req_access = null
 	circuit = /obj/item/circuitboard/machine/clonepod/experimental
 	internal_radio = FALSE
+	var/datum/team/team
```

Comment:
Do you really need a team for this?

Reply:
Idk, I was using owner at first, but then I went with the team cuz it said that owners are deprecated. So be it.

Reply:
Also, the reason I changed to teams is that add_antag_datum actually requires a team to be set, it kinda works without it but it fucks up antag datum it self setting the invalid team value, so to not fuck around with that I again changed to teams.